tests/admin/custom_views/suspend_fraudulent_users_by_ids_test.py

```diff
@@ -0,0 +1,33 @@
+import io
+
+import pytest
+
+from pcapi.core.bookings.factories import BookingFactory
+from pcapi.core.bookings.factories import UsedBookingFactory
+from pcapi.core.users.factories import AdminFactory
+from pcapi.core.users.factories import BeneficiaryGrant18Factory
+
+from tests.conftest import TestClient
+
+
+@pytest.mark.usefixtures("db_session")
+class SuspendFraudulentUsersByIdsTest:
+    def test_suspend_users_by_ids(self, app):
+        admin_user = AdminFactory()
+        fraudulent_user_1 = BeneficiaryGrant18Factory(id=5)
+        fraudulent_user_2 = BeneficiaryGrant18Factory(id=16)
+        BookingFactory(user=fraudulent_user_1)
+        UsedBookingFactory(user=fraudulent_user_2)
+        user_ids_csv = (io.BytesIO(b"user_id\n5\n16"), "user_ids.csv")
+        files = {"user_ids_csv": user_ids_csv}
+        headers = {"content-type": "multipart/form-data"}
+
+        client = TestClient(app.test_client()).with_session_auth(admin_user.email)
```

Comment:
Comme c'est une nouvelle route, c'est l'occasion d'utiliser la fixture `client` récemment ajoutée par Benoît :

```suggestion
        client = client.with_session_auth(admin_user.email)
```

Il faut juste remplacer `app` par `client` dans la signature de `test_suspend_users_by_ids()`.